docs/UserGuide.md

```diff
@@ -145,6 +145,23 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding an order under a Person : `addOrder`
+
+Adds an order under a specified person from the address book.
+
+Format: `addOrder INDEX i/ITEM d/DATE [c/QUANTITY]`
+
+* Add an order under the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index and quantity **must be a positive integer** 1, 2, 3, …​
+* Date must follow the following format: `DD-MM-YYYY`
+* If the quantity is not provided, the quantity will be set to a default value of **1**.
+
+Examples:
+* `addOrder 2 i/Lamp d/20-11-2024 c/3` 
+* `addOrder 1 i/Books d/02-03-2026` Adds the order with item `Books` with quantity of **1** and delivery date `02-03-2026`
```

Comment:
maybe u can say who it adds under? right now the usage is unclear @kaixuan477 help I not good at these i also not sure what is best way.

Reply:
```suggestion
* `addOrder 1 i/Books d/02-03-2026` adds the order with item `Books` with quantity of **1** and delivery date `02-03-2026`, to the first person in the address book.
```